app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html

```diff
@@ -28,19 +29,27 @@
 
 @()(implicit request: Request[_], messages: Messages)
 
-@contactHmrcLink = {
+@registerLink = {
 @link(
-    id = Some("deskpro_link"),
-    text = messages("unauthorised.paragraph.2.link"),
-    call = Call("GET", config.contactHmrcUrl)
-)
+        id = Some("register_for_ppt_link"),
+        text = messages("unauthorised.paragraph.1.link"),
+        call = Call("GET", StartController.displayStartPage().url)
```

Comment:
StartController.displayStartPage() is a Call I believe?

Reply:
```suggestion
        call = StartController.displayStartPage()
```

Reply:
Yup. It is. I can refactor this. Good spot. 👍